swarm/network/kademlia.go

```diff
@@ -536,71 +570,127 @@ func (k *Kademlia) Prune(c <-chan time.Time) {
 	}()
 }
 
+// PeerPot keeps info about expected nearest neighbours and empty bins
+// used for testing only
+type PeerPot struct {
```

Comment:
yes this should be in testing code but since its used by the discovery test external to this pkg it would have to be in swarm/network/testing